File: server/game/paddle.py
from game_core.actor import Actor


class Paddle(Actor):
    speed = None
    score = None

    def __init__(self, actor_id, width=0, height=0):
        super(Paddle, self).__init__(actor_id, width, height)

        self.score = 0

    def init(self, args):
        self.speed = 10

    def act(self):
        pass

File: src/client/lib/game/actors/paddle.dart
import 'package:client/game/core/abstract/actor.dart';

class Paddle extends Actor
{

  int speed;

  Paddle(String ID) : super(ID);
  
  @override
  init([dynamic args=null]) {
    speed = 10;
  }

  @override
  act() {

  }
}